src/routes/transactions/mappers/multisig-transactions/multisig-transaction-execution-info.mapper.spec.ts

```diff
@@ -1,169 +1,140 @@
-    const confirmationsRequired = faker.datatype.number({ min: 0 });
-    const txStatus = TransactionStatus.AwaitingConfirmations;
-    const confirmations = [
-      confirmationBuilder().build(),
-      confirmationBuilder().build(),
-    ];
-    const safeOwners = [
-      faker.finance.ethereumAddress(),
-      faker.finance.ethereumAddress(),
-    ];
-    const safe = safeBuilder().with('owners', safeOwners).build();
-    const transaction = multisigTransactionBuilder()
-      .with('nonce', nonce)
-      .with('confirmations', confirmations)
-      .with('confirmationsRequired', confirmationsRequired)
+    const transaction = multisigTransactionBuilder().build();
+    const safe = safeBuilder()
+      .with('owners', [
+        faker.finance.ethereumAddress(),
+        faker.finance.ethereumAddress(),
+      ])
       .build();
-
-    const executionInfo = mapper.mapExecutionInfo(transaction, safe, txStatus);
+    const executionInfo = mapper.mapExecutionInfo(
+      transaction,
+      safe,
+      TransactionStatus.AwaitingConfirmations,
+    );
 
     expect(executionInfo).toBeInstanceOf(MultisigExecutionInfo);
-    expect(executionInfo).toHaveProperty('nonce', nonce);
+    expect(executionInfo).toHaveProperty('nonce', transaction.nonce);
     expect(executionInfo).toHaveProperty(
       'confirmationsRequired',
-      confirmationsRequired,
+      transaction.confirmationsRequired,
     );
     expect(executionInfo).toHaveProperty(
       'confirmationsSubmitted',
-      confirmations.length,
+      transaction.confirmations?.length,
     );
     expect(executionInfo).toHaveProperty(
       'missingSigners',
-      safeOwners.map((address) => new AddressInfo(address)),
+      safe.owners.map((address) => new AddressInfo(address)),
     );
   });
 
   it('should return a MultiSigExecutionInfo with some safe owners as missing signers', () => {
-    const nonce = faker.datatype.number({ min: 0 });
-    const confirmationsRequired = faker.datatype.number({ min: 0 });
-    const txStatus = TransactionStatus.AwaitingConfirmations;
     const confirmations = [
       confirmationBuilder().build(),
       confirmationBuilder().build(),
     ];
-    const safeOwners = [
-      confirmations[0].owner,
-      faker.finance.ethereumAddress(),
-    ];
-    const safe = safeBuilder().with('owners', safeOwners).build();
     const transaction = multisigTransactionBuilder()
-      .with('nonce', nonce)
       .with('confirmations', confirmations)
-      .with('confirmationsRequired', confirmationsRequired)
+      .build();
+    const safe = safeBuilder()
+      .with('owners', [confirmations[0].owner, faker.finance.ethereumAddress()])
       .build();
 
-    const executionInfo = mapper.mapExecutionInfo(transaction, safe, txStatus);
+    const executionInfo = mapper.mapExecutionInfo(
+      transaction,
+      safe,
+      TransactionStatus.AwaitingConfirmations,
+    );
 
     expect(executionInfo).toBeInstanceOf(MultisigExecutionInfo);
```

Comment:
OOC: why not compare with an instance of `MultisigExecutionInfo`.

I've noticed this in other tests too.

Reply:
Yes, I think it's clearer that way, thanks. Changed in [5f31059](https://github.com/5afe/safe-client-gateway-nest/pull/244/commits/5f31059f0f8ea331058209530f8b73f5ee76432c) 🙂